src/python/grpcio/grpc/_cython/_cygrpc/aio/grpc_aio.pyx.pxi

```diff
@@ -13,16 +13,32 @@
 # limitations under the License.
 
 
-cdef bint _grpc_aio_initialized = 0
+cdef bint _grpc_aio_initialized = False
+# NOTE(lidiz) Theoretically, applications can run in multiple event loops as
+# long as they are in the same thread with same magic. However, I don't think
+# we should support this use case. So, the gRPC Python Async Stack should use
+# a single event loop picked by "init_grpc_aio".
```

Comment:
Not sure about this change, Asyncio runs one loop per thread. If the user spins up a new Thread, hence a new Asyncio loop this will lead to an invalid usage of `call_soon` - explicitly or implicitly because was called behind the scenes by a `create_task`-  when we try to wake up a future that was created in the new loop.

There is an outstanding question about how the driver must work in a multithread and multievent loop environment, and TBH we do not need to give an answer to this right now since I would say that this is a not well spread and used pattern (the multithread and multiloop environments)

Maybe we should create a Github issue and add it into the Alpha, Beta or GA release. I would be tempted to have this solved by the Beta release.

Meanwhile, are we happy about having this pattern that might raise exceptions if the user starts using multiple threads and multiple loops? or should we add a guard for avoiding this pattern and simply raise a `RuntimeError` when it happens?